integration-tests/src/test/java/io/testerra/test/pretest_status/DataProviderTest.java

```diff
@@ -30,7 +30,6 @@
 
     @Test(dataProviderClass = MyDataProvider.class, dataProvider = "throwException")
     public void testT01_interceptCrashedDataProvider() {
```

Comment:
To easily differentiate between a method that _should_ be empty and a stub we forgot to fill, I'd like the former to have this body:
```java
{}
```
but the latter to have this body:
```java
{

}
```
(Basically have the newline signify that _there should be something_ to be delimited by it.)

Reply:
Using the IntelliJ formating it results in 
```
public void testT01_interceptCrashedDataProvider() {
}
```
I don't want to change the defaults.